### PR TITLE
Improve index based commands

### PR DESCRIPTION
**Pull request by khooroko**
Improvements in this PR: delete, edit, borrow, payback

### CODE REVIEW

**Review by jelneo — APPROVED** [review]

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -137,6 +138,10 @@ Examples:
 Edits the phone number, postal code, and email address of the 1st person to be `91234567`, `333111`, and `johndoe@example.com` respectively.
 * `edit 2 n/Betsy Crower t/` +
 Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `list` +
+`select 1` +
+`edit n/Peeta Pen` +
+Edits the name of the 2nd person in the address book to be `Peeta Pen`.
```
Comment (jaivigneshvenugopal) [inline docs/UserGuide.adoc]: 1st person

**Review by lawwman — APPROVED** [review]
Good implementation! Approvedc

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -174,14 +180,19 @@ Examples:
...
+Increases the debt of the 2nd person by $234.
 
 === Decreasing the debt of a person: `payback`
 
 Decrease the debt of a person by the amount entered. +
 Format: `payback INDEX AMOUNT`
 
 ****
-* Decrease the debt of the person at the specified `INDEX` by `AMOUNT`. The index refers to the index number shown in the last person listing. The index must be a positive integer 1, 2, 3, …​
+* Decrease the debt of the person at the specified `INDEX` by `AMOUNT`. The index refers to the index number shown in the last person listing. The index *must be a positive integer* 1, 2, 3, ...
```
Comment (jaivigneshvenugopal) [inline docs/UserGuide.adoc]: Decreases

**Review by jaivigneshvenugopal — APPROVED** [review]
Great job mann!

**Pull request merged**